src/main/webapp/resources/js/components/sample/SampleFileUploader.jsx

```diff
@@ -1,6 +1,7 @@
 import React, { useState } from "react";
 import { setBaseUrl } from "../../utilities/url-utilities";
 import { FileUploader } from "../files/FileUploader";
+import { showNotification } from "../../modules/notifications";
```

Comment:
Unused import, can you remove it (my fault).

Reply:
Removed import in 7a53390